lib/main.dart

```diff
@@ -33,6 +34,8 @@ class IONApp extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    ref.watch(forceUpdateProvider);
```

Comment:
I guess it would be enough to call a method `checkAndUpdateConfig` from the `AppLifecycleObserver` class as it did for the permissions:

  ```
@override
  Widget build(BuildContext context, WidgetRef ref) {
    useOnAppLifecycleStateChange((_, AppLifecycleState current) {
      ref.read(appLifecycleProvider.notifier).newState = current;

      if (current == AppLifecycleState.resumed) {
        ref.read(permissionsProvider.notifier).checkAllPermissions();
      }
    });
```